eq-author/src/App/page/Design/CalculatedSummaryPageEditor/AnswerSelector.js

```diff
@@ -175,13 +180,29 @@ export class UnwrappedAnswerSelector extends Component {
   };
 
   renderAnswers(answers, answerType) {
-    const { section } = this.props.page;
-    const errorValidationMsg = this.props.getValidationError({
+    const unitInconsistencyError = this.props.getValidationError({
       field: "summaryAnswers",
       message: CALCSUM_SUMMARY_ANSWERS_THE_SAME,
     });
 
-    const isInvalid = Boolean(errorValidationMsg);
+    const minOfTwoAnswersError = this.props.getValidationError({
+      field: "summaryAnswers",
+      message: CALCSUM_ANSWER_NOT_SELECTED,
+    });
+
+    const { section } = this.props.page;
+    const unit = answers[0].properties.unit;
+
+    let errorMsg;
+
+    if (unit !== undefined) {
+      errorMsg = buildLabelError(CALCSUM_ANSWER_NOT_SELECTED, `${unit}`, 20, 19);
+    } else {
+    errorMsg = buildLabelError(CALCSUM_ANSWER_NOT_SELECTED, `${answers[0].type}`, 20, 19);
```

Comment:
```suggestion
    errorMsg = buildLabelError(CALCSUM_ANSWER_NOT_SELECTED, `${answers[0].type.toLowerCase()}`, 20, 19);
```

Reply:
yeah - I ewas actually in 2 minds over this - I thought upper case might make it stand out a bit as it's quite a long error message....